### PR TITLE
Add ipRange for the azure service calculation.

### PR DESCRIPTION
**Pull request by abhijitherekar**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "az-svc",
    "component": "gaia",
    "pr-id": "805",
    "commit-sha": "3604f26d95ce002981daa7bd2e88fc4e9035418a",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "backend",
    "pr-id": "1705",
    "commit-sha": "b9a86ba64e8131f6a5a0f4e464a807e1e3ad622d",
    "pipeline": "master"
  }
]
```

File: specs/cloudnode.spec [hunk 1]
```diff
@@ -25,6 +25,7 @@ indexes:
   - vpcid
   - type
   - subtype
+  - cloudtype
```
Comment (rramalinga) [inline specs/cloudnode.spec:28]: Thinking more about this, the `vpcid` is a good enough index to remove other cloudtypes from the query. I think we dont need an extra field for indexing.

File: specs/+cloudendpointdata.spec [hunk 2]
```diff
@@ -121,6 +121,10 @@ attributes:
     - Interface
     - Gateway
     - GatewayLoadBalancer
+    - MicrosoftDBforMySQLServers
+    - MicrosoftDBforMySQLFlexibleServers
+    - MicrosoftDBforPostgreSQLServers
```
Comment (rramalinga) [inline specs/+cloudendpointdata.spec:126]: Any specific reason to keep them as separate service types. Can we just called them as Managed Services here. The reason I am asking is that this is an abstracted field for endpoints. Adding service specific should not be here but should be part of the managed service object itself.

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "az-svc",
    "component": "gaia",
    "pr-id": "805",
    "commit-sha": "43bcfa97de1f9ae2e2543cb6675513b94085b360",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "backend",
    "pr-id": "1705",
    "commit-sha": "17ac738c6ac523ff98265c012e6b8b11d23264ad",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "prismacloud",
    "pr-id": "86",
    "commit-sha": "00a3e0835a841ea3531e36ca5a97312a2bd0dc2a",
    "pipeline": "master"
  }
]
```

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "az-svc",
    "component": "backend",
    "pr-id": "1705",
    "commit-sha": "3ec49dec52efd5f2697f880e92395e38dad8482e",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "prismacloud",
    "pr-id": "86",
    "commit-sha": "00a3e0835a841ea3531e36ca5a97312a2bd0dc2a",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "gaia",
    "pr-id": "805",
    "commit-sha": "43bcfa97de1f9ae2e2543cb6675513b94085b360",
    "pipeline": "master"
  }
]
```

**Review by rramalinga — APPROVED** [review]

**Comment by abhijitherekar** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "az-svc",
    "component": "gaia",
    "pr-id": "805",
    "commit-sha": "43bcfa97de1f9ae2e2543cb6675513b94085b360",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "backend",
    "pr-id": "1705",
    "commit-sha": "454f70d7efd65e36a0c08a25e41eb909361d75e3",
    "pipeline": "master"
  },
  {
    "project": "az-svc",
    "component": "prismacloud",
    "pr-id": "86",
    "commit-sha": "00a3e0835a841ea3531e36ca5a97312a2bd0dc2a",
    "pipeline": "master"
  }
]
```

**Pull request merged**